Okta.Xamarin/Okta.Xamarin.iOS/OktaViewController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Drawing;
+
+using CoreFoundation;
+using UIKit;
+using Foundation;
+using System.Threading.Tasks;
+
+namespace Okta.Xamarin.iOS
+{
+	[Register("UniversalView")]
+	public class UniversalView : UIView
+	{
+		public UniversalView()
+		{
+			Initialize();
+		}
+
+		public UniversalView(RectangleF bounds) : base(bounds)
+		{
+			Initialize();
+		}
+
+		void Initialize()
+		{
+			BackgroundColor = UIColor.Red;
+		}
+	}
+
+	[Register("OktaViewController")]
+	public class OktaViewController : UIViewController
+	{
+		OidcClient client;
+		public OktaViewController()
+		{
+		}
+
+		public override void DidReceiveMemoryWarning()
+		{
+			// Releases the view if it doesn't have a superview.
+			base.DidReceiveMemoryWarning();
+
+			// Release any cached data, images, etc that aren't in use.
+		}
+
+		public OktaConfig OktaConfig { get; set; }
+
+		public override void ViewDidLoad()
+		{
+			base.ViewDidLoad();
+			// Perform any additional setup after loading the view, typically from a nib.
+
+			OktaConfig config = OktaConfig.LoadFromPList("OktaConfig.plist");
+			// TODO: log telemetry data
```

Comment:
Are we tracking user-agent info?

Reply:
When we're ready we will likely integrate the other Sdk's to make those calls when we decide that's appropriate.